src/main/java/seedu/address/logic/commands/MarkPaidCommand.java

```diff
@@ -74,4 +79,12 @@ public String toString() {
                 .add("targetIndex", targetIndex)
                 .toString();
     }
+
+    private Payment calculatePayment(Payment currentPayment, Fees paidFees) {
+        int currentBalance = Integer.parseInt(currentPayment.balance);
+        int amountPaid = Integer.parseInt(paidFees.value);
+
+        Integer finalAmount = currentBalance + amountPaid;
```

Comment:
Maybe another way to do this could be finalAmount = currentBalance - amountPaid? Maybe it will be more intuitive for future developments as well

Reply:
Maybe we could also look at having a fixed amount to be paid with indication of how many months the student currently has to pay for up till the current month